### PR TITLE
[Trivial] inconsistency in runtime arguments

### PR DESCRIPTION
**Pull request by bh2smith**
Not sure how much this matters, but the capitalized name was different with the argument name. Given that neither spellings are actually real words I chose the one I thought looked more appealing.

### CODE REVIEW

**Review by vkgnosis — APPROVED** [review]
Before merging should check that this isn't set on staging or if it is fix the env var there too.

File: solver/src/main.rs
Comment (nlordell) [inline solver/src/main.rs:127]: If you just use `env` it will automatically just use `field_name.to_screaming_snake_case()`.

**Review by nlordell — APPROVED** [review]

File: solver/src/main.rs
Comment (nlordell) [reply solver/src/main.rs:127]: Yes.

Comment (bh2smith) [reply solver/src/main.rs:127]: I could get behind this. I do it right here in this PR if everyone else is onboard?

**Comment by bh2smith** [comment]
> Before merging should check that this isn't set on staging or if it is fix the env var there too.

I can't see anywhere in staging that this is used yet in any of our configuration files. Seems like its fine to change and won't actually require anything for next release.

I am now considering removing all the `env = CAPS_STUFF` everywhere right here, but this also shouldn't require any action for deployment.

File: solver/src/main.rs
Comment (nlordell) [reply solver/src/main.rs:127]: I prefer it this way, but I'm not sure if the `env = "EXPLICIT_NAME"` was intentional or not.

**Review by fleupold — APPROVED** [review]

**Comment by bh2smith** [comment]
Given that this PR has received a lot of approvals and will probably be forgotten. I will leave the changes as is and merge whats here. Will follow up with another PR for removal of `env = CAPS_STUFF` as proposed here.

**Pull request merged**
